plugins/opensslcryptoplugin/opensslcryptoplugin.cpp

```diff
@@ -87,6 +90,19 @@ Daemon::Plugins::OpenSslCryptoPlugin::generateAndStoreKey(
                   QLatin1String("The OpenSSL crypto plugin doesn't support storing keys"));
 }
 
+Result
+Daemon::Plugins::OpenSslCryptoPlugin::importAndStoreKey(
+        const Sailfish::Crypto::Key &key,
+        const QByteArray &passphrase,
+        Key *keyMetadata)
+{
+    Q_UNUSED(key);
+    Q_UNUSED(passphrase);
+    Q_UNUSED(keyMetadata);
+    return Result(Result::UnsupportedOperation,
+                  QLatin1String("The OpenSSL crypto plugin doesn't support storing keys"));
```

Comment:
This should be like `generateStoredKey`, ie. it should be supported when part of the `SqlCipherPlugin`.

Reply:
Ok, I see you implemented it directly in the `SqlCipherPlugin`.